scipy/integrate/_quadrature.py

```diff
@@ -914,15 +914,19 @@ def cumulative_simpson(y, *, x=None, dx=1.0, axis=-1, initial=None):
     Parameters
     ----------
     y : array_like
-        Values to integrate.
+        Values to integrate. Requires at least 1 point along `axis`. If 2 or fewer
+        points are provided along `axis`, Simpson's integration is not possible and the
+        result is calculated with `cumulative_trapezoid`.
     x : array_like, optional
-        The coordinate to integrate along. Must be monotonically increasing.
-        If None (default), use spacing `dx` between consecutive elements in 
-        `y`.
+        The coordinate to integrate along. Must have the same shape as `y` or
+        must be 1D with the same length as `y` along `axis`. x` must also be
+        strictly increasing along `axis`.
+        If `x` is None (default), integration is performed using spacing `dx`
+        between consecutive elements in `y`.
     dx : scalar or array_like, optional
         Spacing between elements of `y`. Only used if `x` is None. Can either 
         be a float, or an array with the same shape as `y`, but length 1 along
-        `axis`.
+        `axis`. Defaults to 1.0.
```

Comment:
```suggestion
        `axis`. Default is 1.0.
```
Not a big deal, but this would be more consistent with the rest of this function's documentation.